lib/active_model/serializer/field.rb

```diff
@@ -0,0 +1,40 @@
+module ActiveModel
+  class Serializer
+    Field = Struct.new(:name, :options, :block) do
```

Comment:
Kept it as a struct because tests in `test/serializers/association_macros_test.rb` made use of the comparability. Wouldn't mind modifying those in a subsequent PR.